src/services/balance.ts

```diff
@@ -37,10 +37,7 @@ export class Balance implements IBalance {
         return;
       }
 
-      const isNegative = balance.startsWith('-');
-      this.bnRaw = new Decimal(
-        (isNegative ? '-0x' : '0x') + balance.replace('-', ''),
-      );
+      this.bnRaw = new Decimal(balance);
```

Comment:
Are we safe here?

Reply:
it was a basic idea. if number starts with 0x - hex, if not - 10